### PR TITLE
carbon-components-react: add hideLabel attribute in DatePickerInputProps

### PR DESCRIPTION
**Pull request by matchilling**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

File: types/carbon-components-react/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Type definitions for carbon-components-react 7.7
 // Project: https://github.com/carbon-design-system/carbon/tree/master/packages/react
 // Definitions by: Kyle Albert <https://github.com/kalbert312>
-//                 Sebastien Gregoire <https://github.com/sgregoire>
```
Comment (sgregoire) [inline types/carbon-components-react/index.d.ts]: Please, do not remove other contributors

**Review by sgregoire — CHANGES_REQUESTED** [review]
Except changes on `types/carbon-components-react/index.d.ts`, seems great.

Next time, please be sure you have added a link to the right [javascript file](https://github.com/carbon-design-system/carbon/blob/v10.7.4/packages/react/src/components/DatePickerInput/DatePickerInput.js#L51).

File: types/carbon-components-react/index.d.ts (hunk 1 above)
Comment (matchilling) [reply types/carbon-components-react/index.d.ts]: Hi @sgregoire , sry 🙏was without intention. Fixed it though.

**Review by sgregoire — APPROVED** [review]
All good, please note that your fork is outdated we are now in version `3.7.x`.

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**